lib/available.fish

```diff
@@ -4,6 +4,6 @@
 # OVERVIEW
 #   Check if a program is available.
 
-function available -a program -d "check if a program is available."
-  type "$program" ^/dev/null >&2
+function available -a name -d "Check if a program is available."
```

Comment:
Leveraging this change, `Check if a function or program is available` describes the command better.